completion/services.py

```diff
@@ -76,6 +76,28 @@ def fill_in_run(block_key):
                 completions[candidate] = 0.0
         return completions
 
+    def get_user_children(self, node):
```

Comment:
I got confused by this method name, because you're actually getting `descendents` or maybe even `leaf_descendents`, since it's not just children being returned, it's doing the full leaf recursion.

Reply:
Yeah, I went with user children because it is really trying to get to the content relevant for a particular user. In the library content case, it wants the problems selected for that user. I try to call this out in the docstring as well, but perhaps I should add it higher in the description.

Reply:
Maybe `get_user_leaf_blocks`? I'm not going to block on that, though.